sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -115,6 +115,40 @@ export function useFormatters() {
     return formattedAddress.join(", ");
   };
 
+  /**
+   * Format an address in the expected order where each expected
+   * address component represents an entry in the array.
+   * @param address address to be formatted.
+   * @returns string array with ordered address components.
+   * @example
+   * [
+   *   "Address Line 1"
+   *   "Address Line 2"
+   *   "City, Province, Postal Code"
+   *   "Country"
+   * ]
+   */
+  const getFormattedAddressList = (address: Address): string[] => {
+    const formattedAddress: string[] = [];
+    // Address line 1
+    formattedAddress.push(address.addressLine1);
+    if (address.addressLine2) {
+      // Address line 2
+      formattedAddress.push(address.addressLine2);
+    }
+    // Country, province, postal code.
+    const cityPostalCountry: string[] = [];
+    cityPostalCountry.push(address.city);
+    if (address.provinceState) {
```

Comment:
I see that `provinceState` is optional in `Address`. Shouldn't it be mandatory?

Reply:
It is mandatory because the addressees outside Canada when we do not capture this information.